api/net/Areas/Admin/Controllers/ActionController.cs

```diff
@@ -86,7 +86,7 @@ public IActionResult FindById(int id)
     [SwaggerOperation(Tags = new[] { "Action" })]
     public IActionResult Add(ActionModel model)
     {
-        var result = _service.Add(model.ToEntity());
+        var result = _service.AddAndSave(model.ToEntity());
```

Comment:
`AddAndSave(...)` replaces the old `Add(...)` methods as the default we've been using.  I've made this change so that we can choose to update multiple entities in a single transaction now.